lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -379,7 +379,8 @@ def build_xml_request
       end
 
       def commit(request)
-        request = "xmldata=#{request}".delete('&')
+        request = "xmldata=#{request}".scrub.delete('&')
```

Comment:
errrr sorry if dumb q but why are we calling `scrub` here when the new method is `truncate`?

Reply:
Update: I figured out what was going on with `truncate` but am still not sure why we're `scrub`bing :D

Reply:
Well, I could have sworn there was a reason I was using `scrub`, but after running the tests without it, I can't remember when and why I added it. Just going to remove it 🤷